### PR TITLE
DSDEGP-1425 Update docker-building logic.

### PR DESCRIPTION
**Pull request by danxmoran**
Based heavily off of the script / Dockerfile in the docker-gradle project.

### CODE REVIEW

File: sbt-list.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# sbt versions to build images for
+0.13.15
```
Comment (danxmoran) [inline sbt-list.cfg:2]: I stuck SBT version in a list too in prep for 1.0.

File: scala-list.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# scala versions to build images for
+2.11.8
```
Comment (hjfbynara) [inline scala-list.cfg:2]: Is there any reason we want the various versions instead of just the latest for this first build

Comment (danxmoran) [reply scala-list.cfg:2]: No particular reason, just didn't see a reason not to cover the bases. Want me to delete all but 2.12.2?

File: docker-build.sh [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      # See if version exists on docker hub
+      docker pull ${REPO}:${docker_version}
+      retcode=$?
+
+      if [ "$retcode" -eq "0" ]; then
+        echo "Skipping version"
+        build_version=0
+      else
+        build_version=1
+      fi
```
Comment (hjfbynara) [inline docker-build.sh]: I think we will need an else clause so setting FORCE=1 will cause all versions to be built regardless if they exist.

else
   build_version=1

**Comment by hjfbynara** [comment]
Couple comments inline the code listing that you should address.

File: docker-build.sh (hunk 3 above)
Comment (danxmoran) [reply docker-build.sh]: Instead of repeating `else build_version=1` twice, I changed `build_version` to default to 1.

**Comment by danxmoran** [comment]
Fixed the if-else but kept the list of Scala versions. Anything else before merge?

**Pull request merged**